easybuild/easyconfigs/s/SIP/SIP-4.18.1-foss-2015a-Python-2.7.11.eb

```diff
@@ -0,0 +1,34 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
```

Comment:
Please fix the URL to use the new `easybuilders` repo address, here and elsewhere.